src/app/components/group/group.component.ts

```diff
@@ -13,6 +13,7 @@ export class GroupComponent implements OnInit {
   @Input() groupId: string;
   @Input() messages: Message[];
   @Input() enabledGroup: string;
+  @Input() allowAdminFunctions; boolean = false;
```

Comment:
Also, is this the right syntax? The ; after allowAdminFunctions seems like it should be a :